pyiron_base/job/script.py

```diff
@@ -31,6 +31,68 @@ class ScriptJob(GenericJob):
         project (ProjectHDFio): ProjectHDFio instance which points to the HDF5 file the job is stored in
         job_name (str): name of the job, which has to be unique within the project
 
+    Example:
+        To submit a notebook as a ScriptJob job to the pyiron job management system,
+
+            1. Create the notebook to be submitted, for ex. 'example.ipynb', and save it -- Can contain any code like:
+                    ```
+                    from pyiron import Project
+                    pr = Project('example')
+                    job = pr.create_job(job_type=pr.job_type.Lammps, job_name='job')  # we name the job 'job'
+                    job.structure = pr.create_ase_bulk('Fe')  # specify structure
+
+                    # Optional: get an input value from 'submit_example_job.ipynb', the notebook which submits
+                    #   'example.ipynb'
+                    external_input = pr.get_external_input()
+                    job.server.cores = external_input['number_of_cores']
+
+                    job.run()  # run the job
+
+                    # save a custom output, that can be used by the notebook 'submit_example_job.ipynb'
+                    job['user/my_custom_output'] = 16
+                    ```
+
+            2. Create the notebook which submits the notebook created in 1., for ex. 'submit_example_job.ipynb',
+                which can have the following code:
+                    ```
+                    from pyiron import Project
```

Comment:
Here it should indeed be `from pyiron`, but `from pyiron.base` might be even more explicit? Not sure if we have the syntax in `pyiron/pyiron` set up that way yet though ;)

Reply:
still, could be `from pyiron_base` safely